corehq/motech/repeaters/expression/repeater_generators.py

```diff
@@ -35,37 +36,25 @@ def get_url(self, repeat_record, url_template, payload_doc):
         }
         return url_template.format(
             **{
-                template_var: expressions[template_var](payload_doc) if template_var in expressions else ""
+                template_var: expressions[template_var](payload_doc_json) if template_var in expressions else ""
                 for template_var in required_template_vars
             }
         )
 
 
-class FormExpressionPayloadGenerator(ExpressionPayloadGenerator):
```

Comment:
The only difference between this and the parent class was to convert the `XFormInstance` to a dict because the case repeater was passing in a dict already. With the change to pass in a `CommCareCase` object this class is no longer needed and the changes have been incorporated into the parent class.